adapters/Vungle/VungleAdapter/Bidding/GADMediationVungleRewardedAd.m

```diff
@@ -143,18 +143,18 @@ - (void)adAvailable {
 
 - (void)didCloseAd {
   [_delegate didDismissFullScreenView];
-
-  [GADMAdapterVungleBiddingRouter.sharedInstance removeDelegate:self];
```

Comment:
Why this is removed?

We still have `addDelegate` in the bidding router:
https://github.com/googleads/googleads-mobile-ios-mediation/blob/master/adapters/Vungle/VungleAdapter/Bidding/GADMAdapterVungleBiddingRouter.m#L73-L81

Reply:
Because this is a duplicate callback. It is handled after the `didCloseAd` call in the router.

https://github.com/googleads/googleads-mobile-ios-mediation/blob/master/adapters/Vungle/VungleAdapter/Bidding/GADMAdapterVungleBiddingRouter.m#L210